docs/docusaurus/docs/guides/expectations/add_features_custom_expectations_lp.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_label: 'Add features to Custom Expectations'
+title: 'Add features to Custom Expectations'
+id: add_features_custom_expectations_lp
+description: Add additional functionality to your Custom Expectations.
+---
+
+import LinkCardGrid from '/docs/components/LinkCardGrid';
+import LinkCard from '/docs/components/LinkCard';
+
+<p class="DocItem__header-description">Add additional functionality to your Custom Expectations.</p>
+
+<LinkCardGrid>
+  <LinkCard topIcon label="Add comments to Expectations and display them in Data Docs" description="Add descriptive comments to Expectations and display them in Data Docs" href="/docs/guides/expectations/advanced/how_to_add_comments_to_expectations_and_display_them_in_data_docs" icon="/img/comment_icon.svg" />
+  <LinkCard topIcon label="Create example cases for a Custom Expectation" description="Add example cases to document and test the behavior of your Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation" icon="/img/example_cases_icon.svg" />
+  <LinkCard topIcon label="Add input validation and type checking to a Custom Expectation" description="Add validation and Type Checking to the input parameters of a Custom Expectation" href="/docs/guides/expectations/features_custom_expectations/how_to_add_input_validation_for_an_expectation" icon="/img/validation_check_icon.svg" />
+  <LinkCard topIcon label="Add Spark support for Custom Expectations" description="Add native Spark support for your Custom Expectations" href="/docs/guides/expectations/features_custom_expectations/how_to_add_spark_support_for_an_expectation" icon="/img/spark_icon.svg" />
```

Comment:
Is there an official spark logo for this card?  I think we use one for the integrations page.